### PR TITLE
Futbol setup

### PR DESCRIPTION
**Pull request by jpasko1112**

### CODE REVIEW

File: lib/game_manager.rb [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
-  def initialize(location, stat_tracker)
-    @location = location
+  def initialize(game_path, stat_tracker)
+    @games = []
     @stat_tracker = stat_tracker
+    create_games(game_path)
+  end
+
+  def create_games(game_path)
+    data = CSV.parse(File.read(game_path), headers: true, header_converters: :symbol)
```
Comment (danieljsauer) [inline lib/game_manager.rb:12]: makes sense, going bottom up instead of top down

File: lib/futbol.rb [hunk 2]
```diff
@@ -1,12 +1,47 @@
...
+
+  attr_reader :league,
               :season
 
   def initialize(locations)
     @games = (CSV.open locations[:games], headers: true, header_converters: :symbol).map { |game| Game.new(game) }
     @league = (CSV.open locations[:leagues], headers: true, header_converters: :symbol).map { |league| League.new(league) }
     @season = (CSV.open locations[:seasons], headers: true, header_converters: :symbol).map { |season| Season.new(season) }
   end
+
```
Comment (danieljsauer) [inline lib/futbol.rb:12]: These were working per rspec tests. May be useful moving forward for sure

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -1,6 +1,29 @@
...
+              :league,
+              :season
 
   def initialize(locations)
-    @game
+    
+    create_managers(locations)
+  end
+
+  def self.from_csv(locations)
```
Comment (danieljsauer) [inline lib/stat_tracker.rb:11]: This makes sense, I am starting to see how the stat tracker will work

**Review by danieljsauer — APPROVED** [review]
Looked over the changes, they are making sense to me

**Review by garussell — APPROVED** [review]

**Pull request merged**